test_pull_requests

```diff
@@ -1006,11 +1103,37 @@ popd
             status = result == 'SUCCESS' ? 'success' : 'failure'
           end
         else
+          # in this case, we're in one of three states:
```

Comment:
what would you prefer? my insane reasoning is comments don't start with capitals, but after a period a sentence should be capital... should I just capitalize everything?